<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Dialog</class>
 <widget class="QDialog" name="Dialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>647</width>
    <height>618</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="QFrame" name="frame">
     <layout class="QGridLayout" name="gridLayout_3">
      <item row="1" column="0">
       <widget class="QGroupBox" name="sweep_groupBox">
        <property name="title">
         <string>Keithley 2400</string>
        </property>
        <layout class="QGridLayout" name="gridLayout">
         <item row="0" column="0">
          <widget class="QCheckBox" name="autorange_sweep_checkBox">
           <property name="text">
            <string>Autorange</string>
           </property>
           <property name="checked">
            <bool>false</bool>
           </property>
          </widget>
         </item>
         <item row="2" column="0">
          <widget class="QLabel" name="label_3">
           <property name="text">
            <string>Manual Range</string>
           </property>
           <property name="wordWrap">
            <bool>true</bool>
           </property>
          </widget>
         </item>
         <item row="2" column="1">
          <widget class="QComboBox" name="manual_range_sweep_comboBox">
           <item>
            <property name="text">
             <string>1 uA</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>10 uA</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>100 uA</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>1 mA</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>10 mA</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>100 mA</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>1 A</string>
            </property>
           </item>
          </widget>
         </item>
         <item row="3" column="1">
          <widget class="QDoubleSpinBox" name="current_compliance_sweep_doubleSpinBox"/>
         </item>
         <item row="1" column="1">
          <widget class="QComboBox" name="autozero_sweep_comboBox">
           <item>
            <property name="text">
             <string>on</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>off</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>once</string>
            </property>
           </item>
          </widget>
         </item>
         <item row="4" column="1">
          <widget class="QDoubleSpinBox" name="nplc_sweep_doubleSpinBox"/>
         </item>
         <item row="4" column="0">
          <widget class="QLabel" name="label_5">
           <property name="text">
            <string>Measurement Speed (NPLC)</string>
           </property>
           <property name="wordWrap">
            <bool>true</bool>
           </property>
          </widget>
         </item>
         <item row="1" column="0">
          <widget class="QLabel" name="label_9">
           <property name="text">
            <string>Autozero</string>
           </property>
          </widget>
         </item>
         <item row="3" column="0">
          <widget class="QLabel" name="label_4">
           <property name="text">
            <string>Current Compliance Limit (A)</string>
           </property>
           <property name="wordWrap">
            <bool>true</bool>
           </property>
          </widget>
         </item>
        </layout>
       </widget>
      </item>
      <item row="0" column="0">
       <widget class="QGroupBox" name="constant_groupBox">
        <property name="font">
         <font>
          <pointsize>8</pointsize>
         </font>
        </property>
        <property name="title">
         <string>Keithley 2400</string>
        </property>
        <layout class="QHBoxLayout" name="horizontalLayout">
         <item>
          <widget class="QLabel" name="label">
           <property name="text">
            <string>Current Compliance Level (A)</string>
           </property>
           <property name="wordWrap">
            <bool>true</bool>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QDoubleSpinBox" name="current_compliance_constant_doubleSpinBox"/>
         </item>
        </layout>
       </widget>
      </item>
      <item row="0" column="1" rowspan="2">
       <widget class="QGroupBox" name="groupBox_3">
        <property name="title">
         <string>Software Settings</string>
        </property>
        <layout class="QGridLayout" name="gridLayout_2">
         <item row="2" column="1">
          <widget class="QDoubleSpinBox" name="v_sweep_step_size_doubleSpinBox"/>
         </item>
         <item row="5" column="2">
          <widget class="QCheckBox" name="return_sweep_checkBox">
           <property name="text">
            <string>Return Sweep</string>
           </property>
          </widget>
         </item>
         <item row="8" column="0" colspan="2">
          <widget class="QPushButton" name="start_pushButton">
           <property name="text">
            <string>Start</string>
           </property>
          </widget>
         </item>
         <item row="4" column="3">
          <widget class="QDoubleSpinBox" name="delay_between_averages_doubleSpinBox"/>
         </item>
         <item row="0" column="1">
          <widget class="QDoubleSpinBox" name="v_sweep_start_doubleSpinBox"/>
         </item>
         <item row="5" column="1">
          <widget class="QDoubleSpinBox" name="first_bias_settle_doubleSpinBox"/>
         </item>
         <item row="5" column="0">
          <widget class="QLabel" name="label_14">
           <property name="text">
            <string>First Bias Settle (ms)</string>
           </property>
           <property name="wordWrap">
            <bool>true</bool>
           </property>
          </widget>
         </item>
         <item row="2" column="2">
          <widget class="QLabel" name="label_10">
           <property name="text">
            <string>Preread Delay (ms)</string>
           </property>
           <property name="wordWrap">
            <bool>true</bool>
           </property>
          </widget>
         </item>
         <item row="0" column="0">
          <widget class="QLabel" name="v_sweep_start_label">
           <property name="text">
            <string>Start</string>
           </property>
           <property name="wordWrap">
            <bool>true</bool>
           </property>
          </widget>
         </item>
         <item row="8" column="2" colspan="2">
          <widget class="QPushButton" name="interrupt_pushButton">
           <property name="text">
            <string>Interrupt</string>
           </property>
          </widget>
         </item>
         <item row="0" column="2">
          <widget class="QLabel" name="v_sweep_finish_label">
           <property name="text">
            <string>Finish</string>
           </property>
           <property name="wordWrap">
            <bool>true</bool>
           </property>
          </widget>
         </item>
         <item row="2" column="3">
          <widget class="QDoubleSpinBox" name="preread_delay_doubleSpinBox"/>
         </item>
         <item row="4" column="2">
          <widget class="QLabel" name="label_12">
           <property name="text">
            <string>Delay Between Averages (ms)</string>
           </property>
           <property name="wordWrap">
            <bool>true</bool>
           </property>
          </widget>
         </item>
         <item row="7" column="1">
          <widget class="QComboBox" name="dimension_comboBox">
           <item>
            <property name="text">
             <string>4000 x 20</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>2000 x 20</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>1000 x 20</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>800 x 20</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>400 x 20</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>200 x 20</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>100 x 20</string>
            </property>
           </item>
          </widget>
         </item>
         <item row="0" column="3">
          <widget class="QDoubleSpinBox" name="v_sweep_finish_doubleSpinBox"/>
         </item>
         <item row="7" column="0">
          <widget class="QLabel" name="label_15">
           <property name="text">
            <string>Dimension</string>
           </property>
          </widget>
         </item>
         <item row="4" column="0">
          <widget class="QLabel" name="label_11">
           <property name="text">
            <string>Software Averages</string>
           </property>
           <property name="wordWrap">
            <bool>true</bool>
           </property>
          </widget>
         </item>
         <item row="2" column="0">
          <widget class="QLabel" name="v_sweep_step_size_label">
           <property name="text">
            <string>Step Size</string>
           </property>
           <property name="wordWrap">
            <bool>true</bool>
           </property>
          </widget>
         </item>
         <item row="4" column="1">
          <widget class="QDoubleSpinBox" name="software_averages_doubleSpinBox"/>
         </item>
         <item row="6" column="0">
          <widget class="QLabel" name="v_constant_label">
           <property name="text">
            <string>V_</string>
           </property>
          </widget>
         </item>
         <item row="6" column="1">
          <widget class="QDoubleSpinBox" name="v_constant_doubleSpinBox"/>
         </item>
         <item row="7" column="2">
          <widget class="QLabel" name="label_6">
           <property name="text">
            <string>Thickness (nm)</string>
           </property>
          </widget>
         </item>
         <item row="7" column="3">
          <widget class="QDoubleSpinBox" name="thickness_doubleSpinBox"/>
         </item>
        </layout>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QFrame" name="frame_2">
     <property name="frameShape">
      <enum>QFrame::StyledPanel</enum>
     </property>
     <property name="frameShadow">
      <enum>QFrame::Raised</enum>
     </property>
     <layout class="QHBoxLayout" name="horizontalLayout_2">
      <item>
       <widget class="QFrame" name="g_graph_container">
        <property name="frameShape">
         <enum>QFrame::StyledPanel</enum>
        </property>
        <property name="frameShadow">
         <enum>QFrame::Raised</enum>
        </property>
        <layout class="QGridLayout" name="gridLayout_4"/>
       </widget>
      </item>
      <item>
       <widget class="QFrame" name="ds_graph_container">
        <property name="frameShape">
         <enum>QFrame::StyledPanel</enum>
        </property>
        <property name="frameShadow">
         <enum>QFrame::Raised</enum>
        </property>
        <layout class="QGridLayout" name="gridLayout_5"/>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <tabstops>
  <tabstop>current_compliance_constant_doubleSpinBox</tabstop>
  <tabstop>autorange_sweep_checkBox</tabstop>
  <tabstop>autozero_sweep_comboBox</tabstop>
  <tabstop>manual_range_sweep_comboBox</tabstop>
  <tabstop>current_compliance_sweep_doubleSpinBox</tabstop>
  <tabstop>nplc_sweep_doubleSpinBox</tabstop>
  <tabstop>v_sweep_start_doubleSpinBox</tabstop>
  <tabstop>v_sweep_finish_doubleSpinBox</tabstop>
  <tabstop>v_sweep_step_size_doubleSpinBox</tabstop>
  <tabstop>preread_delay_doubleSpinBox</tabstop>
  <tabstop>software_averages_doubleSpinBox</tabstop>
  <tabstop>delay_between_averages_doubleSpinBox</tabstop>
  <tabstop>first_bias_settle_doubleSpinBox</tabstop>
  <tabstop>return_sweep_checkBox</tabstop>
  <tabstop>v_constant_doubleSpinBox</tabstop>
  <tabstop>dimension_comboBox</tabstop>
  <tabstop>thickness_doubleSpinBox</tabstop>
  <tabstop>start_pushButton</tabstop>
  <tabstop>interrupt_pushButton</tabstop>
 </tabstops>
 <resources/>
 <connections/>
</ui>
